particle-iot/particle-cli
# bugfix/flags

## Description

Fixes a few bugs - specifically:
* `--page` Flag Does Nothing in `particle library list` Command ([ch#41317](https://app.clubhouse.io/particle/story/41317/page-flag-does-nothing-in-particle-library-list-command))
* `--delay` Flag Does Nothing in `particle get` Command ([ch#41153](https://app.clubhouse.io/particle/story/41153/delay-flag-does-nothing-in-particle-get-command))
* Unhandled Promise When Device is Offline in `particle monitor` Command ([ch#41156](https://app.clubhouse.io/particle/story/41156/unhandled-promise-when-device-is-offline-in-particle-monitor-command))


## How to Test

* compare the output of `particle library list` and `particle library list --page 3` and verify the later actually shows the 3rd page of results
* run `particle get --help` and `particle variable get --help` and note that the `--delay` flag is not longer included
* run `particle monitor <offline_device> <variable>` and verify no error stack is shown
* run `particle monitor <offline_device> <variable>`, wait for a single "Timed out." message, then connect your device - verify that once online your variable's value is returned


## Completeness
